### DISCUSSION TITLE
Application for windows 11 that allows you to group items

### DISCUSSION THREAD

**Discussion by NickVanelli** [microsoft/WindowsAppSDK#3765] [Ideas]
Hello, excuse my ignorance on the subject, but I have a doubt that has bothered me for a long time... I started working with the creation of mobile apps, and on this journey I felt a bit disorganized, so I decided to organize some folders and projects to to make my development cleaner and more productive, by organizing project folders that were on the home screen, along with the numerous shortcuts to programs, I felt the need to make my desktop more minimalist, so the idea was to remove the programs, but group them in a "folder"... but, or I'm not good at researching things of the genre or it doesn't exist, but... I wanted a program that did the same on android that, when you put one app on top of another, it creates a simple folder where you can add more and more apps, I wanted this for windows, but I didn't find anything, there's even one that is ( Taskbar group ), which creates a straight bar with the apps on the taskbar, but it didn't work for me , I want it on the home screen, I don't want to use a common folder because it has access and use from the file explorer, and I just wanted to view the items posted there, I've also seen similar programs like ( fencer ), but it's not exactly what I want, I saw that systems coming from Ubuntu have this native, but I don't want to migrate the system, would anyone know how to tell me if Windows is compatible with this type of function, if something in it currently prevents them from creating something like that?

https://canaltech.com.br/windows/windows-11-pode-ter-novas-pastas-de-apps-para-a-area-de-trabalho-218069/

below is the link with a material that I found when researching the subject, if you know a solution and can share it I would be grateful, and I apologize for the ignorance on the subject. Application for windows 11 that allows you to group programs on the desktop, the items would be in a square window with the items placed there.

**Comment by gautambjain** [comment]
Did you try "Stardock Fences"?

**Reply by NickVanelli** [reply]
yes, but as I said, it didn't work for me, the way it works, there are several similar ones, a bar with a name, which opens vertically, at the start of windows, it has a way to group applications, and when opened, it only displays the apps and the group name, that's what I want, only on the desktop.
